src/modules/design/diagram-tools-right/diagram-tools-right.html

```diff
@@ -30,6 +30,12 @@
     <a class="button tool" class.bind="distributeElementsEnabled ? '' : 'button--disabled tool--disabled'" title.bind="distributeElementsEnabled ? 'Distribute Elements Horizontally' : 'Select at least three elements to distribute them horizontally'" click.delegate="distributeElementsHorizontally()" disabled.bind="!distributeElementsEnabled">
       <i class="fas fa-arrows-alt-h"></i>
     </a>
+    <a class="button tool" title="Fit diagram to viewport" click.delegate="fitDiagramToViewport()">
+      <i class="fas fa-expand"></i>
+    </a>
+    <a class="button tool" title="Move viewport to zero point" click.delegate="moveViewportToZeroPoint()">
```

Comment:
Would use `origin` instead of `zero point`. The user does not know coordinates (there is no grid).

`0` as an icon is confusing in the same way. Maybe something with a map marker or [arrows pointing inwards](https://icon-icons.com/icon/four-arrows-directed-to-circle-at-the-center/70727)?
Arrow pointing outwards means _move pane_ and this is in a way the natural opposite.

Reply:
```suggestion
    <a class="button tool" title="Move viewport to origin" click.delegate="moveViewportToZeroPoint()">
```

The problem is that I could not find a good [font awesome](https://fontawesome.com/icons?d=gallery) icon for this case